lib/billing_facade_client.rb

```diff
@@ -48,12 +50,18 @@ def self.get_unit_price(cost_code, product_name)
     end
   end
 
+  def self.get_sub_cost_codes(cost_code)
+    r = connection.get("/accounts/#{cost_code}/subaccountcodes")
+    response = JSON.parse(r.body, symbolize_names: true)
+    return response[:subCostCodes]
+  end
+
   def self.validate_product_name?(product_name)
     validate_single_value("/products/#{product_name}/verify")
   end
 
   def self.validate_cost_code?(cost_code)
-    validate_single_value("/accounts/#{cost_code}/verify")
+    validate_single_value("/subaccountcodes/#{cost_code}/verify")
```

Comment:
It's probably not worth changing the method name, but this now validates sub cost codes instead of cost codes, maybe worth adding a comment to say that? (Unless it is reasonable to change the method name)